docs/dev/static_libaries.md

```diff
@@ -0,0 +1,152 @@
+
+* Build OpenVINO Runtime as usual with the CMake option of `-DSELECTIVE_BUILD=COLLECT`.
+* Run target applications on target models and target platforms to collect traces.
+* Build the final OpenVINO static Runtime with `-DSELECTIVE_BUILD=ON -DSELECTIVE_BUILD_STAT=/path/*.csv -DBUILD_SHARED_LIBS=OFF`
+
+## Building with static MSVC Runtime
+
+In order to build with static MSVC runtime, use the special [OpenVINO toolchain](https://github.com/openvinotoolkit/openvino/blob/master/cmake/toolchains/mt.runtime.win32.toolchain.cmake) file:
+
+```sh
+cmake -DCMAKE_TOOLCHAIN_FILE=<openvino source dir>/cmake/toolchains/mt.runtime.win32.toolchain.cmake <other options>
+```
+
+> **NOTE**: all other dependent application and libraries must be built with the same `mt.runtime.win32.toolchain.cmake ` toolchain to have conformed values of the `MSVC_RUNTIME_LIBRARY` target property.
+
+## Limitations
+
+* CMake version 3.17 or higher must be used to build static OpenVINO libraries.
+* Supported OSes:
+    * Windows x64
+    * Linux x64
+    * All other OSes may work, but have not been explicitly tested
+* The enabled and tested capabilities of OpenVINO Runtime in a static build:
+    * OpenVINO common runtime - work with `ov::Model`, perform model loading on particular device
+    * CPU and GNA inference plugins (**GPU and MYRIAD are not enabled**)
+    * MULTI, HETERO, AUTO, and BATCH inference modes
+    * IR, ONNX, PDPD, and TF frontends to read `ov::Model`
+* Static build support for building static libraries only for OpenVINO Runtime libraries. All other third-party prebuilt dependencies remain in the same format:
+    * `libGNA` is a shared library.
+    * `TBB` is a shared library; to provide your own TBB build from [[oneTBB source code|https://github.com/oneapi-src/oneTBB]] use `export TBBROOT=<tbb_root>` before OpenVINO CMake scripts are run.
+
+    > **NOTE**: The TBB team does not recommend using oneTBB as a static library, see [[Why onetbb does not like a static library?|https://github.com/oneapi-src/oneTBB/issues/646]]
+
+* `TBBBind_2_5` is not available on Windows x64 during a static OpenVINO build (see description for `ENABLE_TBBBIND_2_5` CMake option [[here|CMakeOptionsForCustomCompilation]] to understand what this library is responsible for). So, capabilities enabled by `TBBBind_2_5` are not available. To enable them, build [[oneTBB from source code|https://github.com/oneapi-src/oneTBB]] and provide the path to built oneTBB artifacts via `TBBROOT` environment variable before OpenVINO CMake scripts are run.
+
+* `ov::Op::type_info` static member is deprecated and not available in static build. Don't use `type_info` during implementation of your own custom operations, use `ov::Op::get_type_info_static()` instead. 
+
+## Additional Resources
+
+ * [OpenVINO README](../../README.md)
+ * [OpenVINO Developer Documentation](index.md)
+
```

Comment:
Please add a link to how to build document

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/16063/commits/562c6c32912eb936d9ca384c7caa9cf772ce2fa2